opensrp-chw-referral/src/main/java/org/smartregister/chw/referral/model/AbstractIssueReferralModel.java

```diff
@@ -0,0 +1,92 @@
+package org.smartregister.chw.referral.model;
+
+import android.arch.lifecycle.ViewModel;
+import android.databinding.ObservableField;
+
+import com.google.gson.Gson;
+
+import org.smartregister.chw.referral.contract.BaseIssueReferralContract;
+import org.smartregister.chw.referral.domain.MemberObject;
+import org.smartregister.chw.referral.domain.ReferralServiceIndicatorObject;
+import org.smartregister.chw.referral.domain.ReferralServiceObject;
+import org.smartregister.chw.referral.util.Constants;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+
+import timber.log.Timber;
+
+public abstract class AbstractIssueReferralModel extends ViewModel implements BaseIssueReferralContract.Model {
+    public MemberObject MEMBER_OBJECT;
+    public ObservableField<ReferralServiceObject> selectedReferralService = new ObservableField<>();
+    public List<ReferralServiceIndicatorObject> referralServiceIndicators;
+    public long appointmentDateTimestamp;
+    public String referralFacilityUuid;
```

Comment:
@cozej4 Any reasons for using ```public ``` scope on these fields in an abstract class. Would reducing the scope to ```protected``` have any effects?

Reply:
Yeah, reducing the scope would have an impact, these fields are called elsewhere using the class's or extended classe's objects.